app/index.html

```diff
@@ -18,9 +18,14 @@
 	<i></i>
 </div>
 <script>
-	document.fonts && document.fonts.ready && document.fonts.ready.then(function () {
+	if (localStorage.getItem('hasRubikFont')) {
```

Comment:
someone (@bkoval or @latata) said that cookie would be better for some reason. Don't remember the exact reason but I remember that it seemed to be a valid argument for me. Please consult it

Reply:
also localStorage can throw if user is in incognito mode
also this means that if a browser chose to invalidate cache
we will show blank page to a user for a bit

Reply:
on yet another hand - google caches this fonts for pretty much eternity
"public, max-age=31536000"